test/e2e/sagemaker/resources/kmeans_processingjob.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: ProcessingJob
+metadata:
+  name: $PROCESSING_JOB_NAME
+spec:
+  processingJobName: $PROCESSING_JOB_NAME
+  roleARN: $SAGEMAKER_EXECUTION_ROLE_ARN
+  environment:
+    my_var: my_value
+    my_var2: my_value2
+  appSpecification:
+    imageURI: $PYTORCH_TRAIN_IMAGE_URI
+    containerEntrypoint:
+      - python
+      - /opt/ml/processing/code/kmeans_preprocessing.py
+  networkConfig:
+    enableNetworkIsolation: false
```

Comment:
Were all of these fields always required in the spec file (with the older operators I mean ?)

Reply:
I'm not sure. I checked the file history and this particular line has been in there since day 1. I might actually experiment with removing this, I think this is the only unnecessary line.